tests/v2/validation/nodescaling/scaling_nodepools.go

```diff
@@ -0,0 +1,50 @@
+package scaling
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/machinepools"
+	"github.com/rancher/rancher/tests/framework/extensions/provisioninginput"
+	rke1 "github.com/rancher/rancher/tests/framework/extensions/rke1/nodepools"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads/pods"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	ProvisioningSteveResourceType = "provisioning.cattle.io.cluster"
+	defaultNamespace              = "fleet-default"
+)
+
+func ScalingRKE2K3SNodePools(t *testing.T, client *rancher.Client, clusterID string, clustersConfig *provisioninginput.Config) {
+	cluster, err := client.Steve.SteveType(ProvisioningSteveResourceType).ByID(clusterID)
+	require.NoError(t, err)
+
+	clusterResp, err := machinepools.ScaleMachinePoolNodes(client, cluster, clustersConfig.NodesAndRoles[0])
```

Comment:
why are you indexing from zero? Why not just pass the single node role? I think that makes this much safer, than forcing to use the clustersConfig.